pkg/graph/schema.graphql

```diff
@@ -745,6 +745,8 @@ type SystemIntake {
   createdAt: Time # TODO - This should probably not be nullable, but some data in IMPL & PROD has it nulled out. We should fix this in the future. (see EASI-3090)
   currentStage: String
   decisionNextSteps: HTML
+  eaCollaborator: String
```

Comment:
it might be worthwhile to add a comment that these are only for historic data here with GQL comment syntax so it gets documented
ex
"""
this stores legacy ea collaborator information. We no longer collect it, but display it for historic requests
"""